backend/src/lib/server-side-events.ts

```diff
@@ -274,25 +286,78 @@ export class ServerSideEvents {
       logger.info({ msg: 'event stream close' })
     })
 
-    let lastEventID = 0
-    if (req.headers['last-event-id']) {
-      const last = Number(req.headers['last-event-id'])
-      if (Number.isInteger(last)) {
-        const cookies = parseCookies(req)
-        if (cookies['watch'] === instanceID) {
-          lastEventID = last
-        }
```

Comment:
Need to do some research into this. I'm a bit concerned that by removing this we may be losing the ability for the event stream to reconnect. But it's hard to say how often that happens or how robust it is.